tools/packaging/release/release.sh

```diff
@@ -34,6 +34,10 @@ function _check_required_env_var()
 	local env_var
 
 	case ${1} in
+		RELEASE_VERSION) env_var="${RELEASE_VERSION}" ;;
+		GH_TOKEN) env_var="${GH_TOKEN}" ;;
+		ARCHITECURE) env_var="${ARCHITECURE}" ;;
```

Comment:
`s/ARCHITECURE/ARCHITECTURE`

Unless we're passed a var with a typo?

Reply:
Fixed in this not so small new series.